Plugins/Reqnroll.SpecFlowCompatibility.Generator.ReqnrollPlugin/PackageReadme.md

```diff
@@ -0,0 +1,29 @@
+﻿Reqnroll.SpecFlowCompatibility (pronounced as [reknroʊl]) provides compatibility layer for migrating from SpecFlow to Reqnroll.
+
+This package helps ease the migration from SpecFlow by providing compatibility shims and adapters, allowing you to gradually migrate your existing SpecFlow tests to Reqnroll.
```

Comment:
Add a link to https://docs.reqnroll.net/latest/guides/migrating-from-specflow.html